modin/core/execution/ray/common/task_wrapper.py

```diff
@@ -18,11 +18,12 @@
 """
 
 import asyncio
-
+from modin.logging import ClassLogger, enable_remote_logging
 import ray
 
 
-@ray.remote
+@ray.remote(num_returns=2)
```

Comment:
why change?.. also, as this is a general wrapper, I don't think we can know upfront how many items it would return